src/ariel-os-alloc/src/lib.rs

```diff
@@ -0,0 +1,97 @@
+//! This crate glues `embedded-alloc` into Ariel OS.
```

Comment:
```suggestion
//! This crate glues a suitable into Ariel OS.
```

or

```suggestion
//! This crate glues a suitable into Ariel OS. Currently, this always uses `embedded-alloc`.
```

I don't think we should make the chosen allocator too visible here; we might add features to guide the allocator selection later.

Reply:
I went with "suitable" as this is already using `esp-alloc` on esp.